bin/diff_results

```diff
@@ -48,9 +48,15 @@ import os
 import sys
 
 # R packages are stored relative to the top-level of the repo.
-if ('R_LIBS_USER' not in os.environ or 'rlibs' not in os.environ['R_LIBS_USER']):
-    os.environ['R_LIBS_USER'] = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
-                                             'work', 'rlibs')
+our_rlibs = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), 'work', 'rlibs')
```

Comment:
Can any of this stuff be put in a function for re-usability?

Reply:
There are only two instances, so I'm not too worried.